components/Comments.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+
+import { useState, ChangeEvent, FormEvent, useContext, useEffect } from "react";
+import UserInfoContext from "context/UserInfoContext";
+
+interface Comment {
+  email: string;
+  text: string;
+}
+
+export const Comments = () => {
+  const [comment, setComment] = useState<string>("");
+  const [error, setError] = useState<string>("");
+  const [showComments, setShowComments] = useState<boolean>(false);
+  const [commentsList, setCommentsList] = useState<Comment[]>([]);
+  const [successMessage, setSuccessMessage] = useState<string>("");
+
+  const { userInfo } = useContext(UserInfoContext);
+  const userEmail = userInfo?.email;
+
+  useEffect(() => {
+    if (successMessage) {
+      const timer = setTimeout(() => {
+        setSuccessMessage("");
+      }, 3000);
```

Comment:
Why we have 3000s here, this should be done with promise,when the promise is resolved, then we setSuccessMessage("")

we need to change that